### ISSUE TITLE
Deprecation warning: scikit-learn

### ISSUE THREAD

**Issue by braymp** [CellProfiler/CellProfiler-Analyst#110]
Opening CPA from master using scikit-learn 0.17 gives this this warning:
```
C:\Python27\lib\site-packages\sklearn\lda.py:4: DeprecationWarning: lda.LDA has been moved to discriminant_analysis.LinearDiscriminantAnalysis in 0.17 and will be removed in 0.19
  "in 0.17 and will be removed in 0.19", DeprecationWarning)
C:\Python27\lib\site-packages\sklearn\qda.py:4: DeprecationWarning: qda.QDA has been moved to discriminant_analysis.QuadraticDiscriminantAnalysis in 0.17 and will be removed in 0.19.
  "in 0.17 and will be removed in 0.19.", DeprecationWarning)
```

**Comment by daviddao** [comment]
Fixed in 34701d1
